pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -491,6 +497,9 @@ func configMapDeleted(obj interface{}) {
 		if getSvFssCRAvailability() {
 			log.Debugf("Ignoring supervisor FSS configmap delete event as %q CR is present", featurestates.CRDSingular)
 			return
+		} else if serviceMode == "node" {
```

Comment:
I think we do not need to check the availability of CR as well by calling `getSvFssCRAvailability` when `serviceMode` is node.

Reply:
Availability of CR check is to ignore updates to configmap in controllers after the CR is initialized. This check to see if service mode is node is to ignore updates to configmap in nodes as we are not using this configmap anymore

Reply:
Can we do the following to make it clear that we do not care about sv fss config map deletion/addition and update when service mode is `node`.

```
if fssConfigMap.Name == k8sOrchestratorInstance.supervisorFSS.configMapName &&
		fssConfigMap.Namespace == k8sOrchestratorInstance.supervisorFSS.configMapNamespace {
		if serviceMode == "node" {
			return
		}
		if getSvFssCRAvailability() {
			log.Debugf("Ignoring supervisor FSS configmap delete event as %q CR is present", featurestates.CRDSingular)
			return
		}
		log.Errorf("configMap %q in namespace %q deleted. This is a system resource, kindly restore it.", fssConfigMap.Name, fssConfigMap.Namespace)
		os.Exit(1)
	}
```

Reply:
Done